examples/js/loaders/deprecated/LegacyGLTFLoader.js

```diff
@@ -1634,6 +1634,20 @@ THREE.LegacyGLTFLoader = ( function () {
 									geometry.addAttribute( 'skinIndex', bufferAttribute );
 									break;
 
+								default:
+									var material = json.materials[ primitive.material ];
+									var parameters = json.techniques[ material.technique ].parameters;
+									
+									for( var attributeName in parameters ) {
+
+										if( parameters[attributeName]['semantic'] === attributeId ) {
+
+											geometry.addAttribute( attributeName, bufferAttribute );
```

Comment:
Would just add a `break;` after this line as well.

Reply:
Ah, technically I guess it is possible two parameters might use the same semantic. OK to leave out the `break`.